source/nodejs/adaptivecards-templating/package-lock.json

```diff
@@ -1,9 +1,14 @@
 {
 	"name": "adaptivecards-templating",
-	"version": "0.1.0-alpha.1",
+	"version": "0.1.1-alpha.1",
 	"lockfileVersion": 1,
 	"requires": true,
 	"dependencies": {
+		"@microsoft/recognizers-text-data-types-timex-expression": {
+			"version": "1.3.0",
+			"resolved": "https://botbuilder.myget.org/F/botbuilder-v4-js-daily/npm/@microsoft/recognizers-text-data-types-timex-expression/-/@microsoft/recognizers-text-data-types-timex-expression-1.3.0.tgz",
+			"integrity": "sha1-/F1YbIJuR46Ed7f8sh6eKDDoHGc="
```

Comment:
How will the code change, depending on our discussion outcome on how we want to ship this (whether update to the templating package or update to the rendering package)? Trying to understand whether this should go in as is or if we should reissue the PR once that decision is made (while continuing to iterate on this branch/PR with actual integration changes) ..

Reply:
Either the code will remain in the adaptivecards-templating package, or it will be moved to the adaptivecards package. If the latter I will have to do that work.